driver/src/test/java/org/neo4j/driver/integration/ErrorIT.java

```diff
@@ -52,7 +52,7 @@ public void shouldThrowHelpfulSyntaxError() throws Throwable
     }
 
     @Test
-    public void shouldNotAllowUsingTransactionAfterError() throws Throwable
+    public void shouldNotAllowUsingTransactionAfterRecoverableError() throws Throwable
```

Comment:
Is this name the right way round? A transaction should surely be _usable_ after a _recoverable_ error but _unusable_ after an _unrecoverable_ one...